src/bro_maker/from_files.rs

```diff
@@ -0,0 +1,39 @@
+extern crate yaml_rust;
```

Comment:
`extern crate x` is deprecated syntax, I think you can remove it.

By the way, `yaml_rust` seems rather old, I'd use `serde_yaml` instead, seems more complete and even easier to use:
```rust
let raw_yaml = r#"
name: "John Doe"
age: 30
"#;
let parsed_yaml: HashMap<String, String> = serde_yaml::from_str(raw_yaml).unwrap();
 // none of this "doc[0]" weirdness, parsed_yaml is now a HashMap and you can access elements with keys like you'd expect
println!("{}", parsed_yaml["name"]);
// outputs "John Doe"
```

Reply:
Awesome.

Reply:
My problem with serde was the Derive spam, you solved it for me :)